spec/support/travel_advice_publisher_navigation_helpers.rb

```diff
@@ -0,0 +1,89 @@
+module TravelAdvicePublisherNavigationHelpers
+  def visit_travel_advice_publisher_homepage
+    travel_advice_publisher_url("/admin")
+  end
+
+  def visit_draft_origin_url(path)
+    visit(Plek.find("draft-origin") + path)
+  end
+
+  def visit_published_url(path)
+    visit(Plek.find("www") + path)
+  end
+
+  def travel_advice_publisher_url(path)
```

Comment:
I think this should be called `visit_travel_advice_publisher` and then you could have a separate method which constructs and returns only the URL. And similar to the methods above.